src/modules/settings/components/Settings.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/extensions */
+import React from 'react';
+import './store/*';
+import Panel from 'rsuite/lib/Panel/index.js';
+import Icon from 'rsuite/lib/Icon/index.js';
+import InputGroup from 'rsuite/lib/InputGroup/index.js';
+import AutoComplete from 'rsuite/lib/AutoComplete/index.js';
+import enhance from '../../../assets/icons/search-solid.svg';
+import {Components} from './Store.jsx';
+
+const settings = Object.values(Components).sort((a, b) => a.settingId.localeCompare(b.settingId));
+const AUTOCOMPLETE = settings.map((setting) => setting.autoComplete);
+
+export function Settings({search, category}) {
+  const searchedSettings =
+    search.length === 0
+      ? settings.filter((setting) => setting.category === category).map((setting) => setting.render())
+      : settings
+          .filter((setting) => setting.keywords.join(' ').includes(search.toLowerCase()))
+          .map((setting) => setting.render());
+
+  return searchedSettings.length === 0 ? <Panel header="No more results..." /> : searchedSettings;
+}
+
+export function Search({value, onChange}) {
+  return (
+    <InputGroup>
+      <AutoComplete
+        value={value}
+        data={AUTOCOMPLETE.filter((res) => res.toLowerCase().includes(value)).slice(0, 5)}
```

Comment:
it looks like we support autocomplete and keywords now. will people actually press enter to search if they are getting autocomplete results? do we need to support both in autocomplete?

Reply:
I think we can alternatively, remove the autocomplete attribute and make it only support keywords as the keywords usually include the autocomplete value anyway

Reply:
Scrub that, I think we'll display w.e autocomplete is and filter by the keywords